src/core/updateWebpackConfig.js

```diff
@@ -28,7 +28,11 @@ const updateWebpackConfig = (webpackConfig, env, serverInfo) => {
     }
   )
 
-  webpackConfig.entry.push(stylesPath('styles.css'))
+  webpackConfig.entry.push(
+    stylesPath('styles.css'),
+    'codemirror/lib/codemirror.css',
+    'rsg-codemirror-theme.css'
```

Comment:
do we need to import `codemirror` related css in `snapguidist`? Aren't they already loaded by `react-styleguidist`?
Removing the imports doesn't change the visual output